classes/help.php

```diff
@@ -71,64 +70,64 @@ function print_page() {
 															show_categories_filter=false show_tags_filter=false]</code>
 													</td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want to hide the map, insert shortcode like this:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want to hide the map, insert shortcode like this:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															hide_map=true]</code></td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want to hide the list of results, insert shortcode like this:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want to hide the list of results, insert shortcode like this:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															hide_list=true]</code></td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want to override the default lat / lng for a specific map, insert shortcode like this:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want to override the default lat / lng for a specific map, insert shortcode like this:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															default_lat='34.1346702' default_lng='-118.4389877']</code>
 													</td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'You can combine tag attributes as needed:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'You can combine tag attributes as needed:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															categories=2,5,14 show_tags_filter=false]</code></td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'You can place content above or below your map, just like in any other post. Note that any content placed below the map will be pushed down by the list of search results (unless you have them displaying differently with a custom theme).', 'SimpleMap' ); // XSS ok. ?></td>
+													<td><?php _e( 'You can place content above or below your map, just like in any other post. Note that any content placed below the map will be pushed down by the list of search results (unless you have them displaying differently with a custom theme).', 'simplemap' ); // XSS ok. ?></td>
 												</tr>
 
 												<tr>
-													<td><?php printf( __( 'Configure the appearance of your map on the %s General Options page.%s', 'SimpleMap' ), '<a href="' . esc_url( admin_url( 'admin.php?page=simplemap' ) ) . '">', '</a>' ); ?></td>
+													<td><?php printf( __( 'Configure the appearance of your map on the %s General Options page.%s', 'simplemap' ), '<a href="' . esc_url( admin_url( 'admin.php?page=simplemap' ) ) . '">', '</a>' ); ?></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 253 characters

<!-- Comment by [SideCI](https://sideci.com) -->